### DISCUSSION TITLE
v6200 (Lots of stuff)

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#157] [General]
This is a big version. I'll describe some of the changes:

More compact settings, but also less compact settings (no mode specific versions)

Added more Show settings to hide the top buttons of the interface.

Added `Show Pins` and `Show Unloaded`, which can be unchecked. Added `Toggle Unloaded`.

Filter Enter is a setting to define if Enter is required when filtering normal (tabs, closed), or search (history, bookmarks), or both, or none.

Added a simple system to override default settings. The idea is to provide a file to define all setting default values to be overriden, as to create distros of grasshopper.

https://github.com/madprops/grasshopper/blob/main/js/overrides.js

Lots of changes in Triggers. Added mode specific click events and others.

Removed settings `Load Lock` and `Click Select` because the configurations are possible by tweaking trigger settings.

Lots of internal Menus modifications to bring names to current standards. Might need to edit backups.

Added `Soft Action` command which is an Action command (click on item) except it won't load unloaded items.

Also added `Select Item` command, which only selects items.

These are meant to be used on Trigger mappings.

![showmods](https://github.com/user-attachments/assets/078ff5cb-1865-4af9-8cc8-5a214e451c02)

![triggers2345](https://github.com/user-attachments/assets/153a2571-6b0a-4c2f-9f35-f0bb9e0caffb)

**Comment by N3C2L** [comment]
Thanks for this great stuff!

**Comment by madprops** [comment]
Also adding mode-specific hover menus.

![overhj234](https://github.com/user-attachments/assets/c56dff38-1ae0-4123-8da3-1c7c5cacb304)
